src/App.css

```diff
@@ -1,39 +1,194 @@
-.App {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+  background-color: gainsboro;
+}
+
+.left {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+.container-center {
+  max-width: 1000px;
+  margin: 0 auto;
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+li {
+  list-style-type: none;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+header {
+  background-color: #ffff;
+  height: 80px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+nav {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px 0;
+}
```

Comment:
Great design @katfogy 🤝

- [ ] As stated in the project requirements, kindly do not repeat the same rule for multiple styles. Instead, create a single class and reuse this class instead. You may create classes for the following repeating rules which include. `background-color: #ffff;`, `display: flex;`, and `list-style-type: none;`.